doc/manual/src/store/file-system-object/content-address.md

```diff
@@ -0,0 +1,83 @@
+# Content-Addressing File System Objects
+
+For many operations, Nix needs to calculate [a content addresses](@docroot@/glossary.md#gloss-content-address) of [a file system object][file system object].
+Usually this is needed as part of content addressing [store objects], since store objects always have a root file system object.
+But some command-line utilities also just work on "raw" file system objects, not part of any store object.
+
+Every content addressing scheme Nix uses ultimately involves feeding data into a [hash function](https://en.wikipedia.org/wiki/Hash_function), and getting back an opaque fixed-size digest which is deemed a content address.
+The various *methods* of content addressing thus differ in how abstract data (in this case, a file system object and its descendents) are fed into the hash function.
+
+## Serialising File System Objects { #serial }
+
+The simplest method is to serialise the entire file system object tree into a single binary string, and then hash that binary string, yielding the content address.
+In this section we describe the extent methods of serialising file system objects.
```

Comment:
I don't understand "extent methods". Could this be a typo?

Reply:
Oh I just meant the ones that currently exist, letting slip I want more of them hehe :)
```suggestion
In this section we describe the currently-supported methods of serialising file system objects.
```
```suggestion
In this section we describe the supported methods of serialising file system objects.
```
```suggestion
In this section we describe the methods of serialising file system objects.
```
you can pick!

Reply:
It won't let me!

Let's try this:

```suggestion
In this section we describe the extent methods of serialising file system objects.
```

Reply:
Nope. I'm in "all commits" mode, but it won't let me. Could you amend it?

Reply:
Oh, right, this branch is organization-owned.

Reply:
Just say which one you want then?

Reply:
(Your last suggestion was a no op, so I don't know which you picked.)